### PR TITLE
add explanation of working in a text editor and how to run files in p…

### PR DESCRIPTION
**Pull request by KarinRoe**
…ython

### CODE REVIEW

File: python/lesson1/tutorial.md [hunk 1]
```diff
@@ -121,6 +124,30 @@ multiple things at once, separated by a comma:
...
+You can read your file with the following command (type without the $ sign):
+
+	$ python ex1.py
+
+If you have `>>>`in front of your code, you are still in REPL(IDLE) and need to exit
+it with:
+
+	>>> quit()
+
+Than you should be able to load the file.
```
Comment (brunogirin) [inline python/lesson1/tutorial.md]:
```suggestion
Then you should be able to load the file. 
```

File: python/lesson1/tutorial.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ title: Strings, Integers and Floats
 In this tutorial, we're going to start to work with the basic types of Python:
 strings (for text) and integers & floats (for numeric values).
 
+Notice that in this tutorial you are working in REPL (IDLE). You find more information
```
Comment (brunogirin) [inline python/lesson1/tutorial.md]:
```suggestion
Notice that in this tutorial you are working in REPL (IDLE). You can find more information
```

File: python/lesson1/tutorial.md [hunk 3]
```diff
@@ -121,6 +124,30 @@ multiple things at once, separated by a comma:
 
     >>> print('The result of 2 + 2 is', 2 + 2)
 
+### Saving Your Work
+
+In this tutorial you coded in the REPL (IDLE), but a lot of times you want
+to save your code instead. In these casees you would use a text editor.
```
Comment (brunogirin) [inline python/lesson1/tutorial.md]:
```suggestion
to save your code instead. In such cases you can save your code to a file using a text editor. 
```

**Comment by KimberleyCook** [comment]
Thanks for your contribution @KarinRoe :D 

And thanks for reviewing @brunogirin, are you happy for me to merge?

**Comment by brunogirin** [comment]
@KimberleyCook sorry for not coming back to you, yes I'm happy for you to merge!

**Review by brunogirin — APPROVED** [review]

**Review by KimberleyCook — APPROVED** [review]

**Pull request merged**